client/src/components/Balance/BalanceTransfer.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+import CurrentBalance from './CurrentBalance.jsx';
+import BalanceAPIutils from './BalanceAPIutils'
+
+function BalanceTransfer({ userId }) {
```

Comment:
i wonder what the tradeoffs are between using a functional or class component when there's a lot of state involved? i default to class components when handling a lot of state, but i also haven't practiced using functional components/hooks very much.